browser/brave_rewards/rewards_panel_helper.cc

```diff
@@ -0,0 +1,56 @@
+/* Copyright (c) 2021 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "brave/browser/brave_rewards/rewards_panel_helper.h"
+
+#include <memory>
+#include <string>
+#include <utility>
+
+#include "brave/browser/ui/views/brave_actions/brave_actions_container.h"
+#include "chrome/browser/profiles/profile.h"
+#include "chrome/browser/ui/browser_finder.h"
+#include "extensions/buildflags/buildflags.h"
+
+namespace {
+constexpr char kAdaptiveCaptchaPanelUrl[] = "adaptive_captcha_panel.html";
+constexpr char kRewardsPanelUrl[] = "brave_rewards_panel.html";
+
+bool ShowPanel(content::BrowserContext* context,
+               const std::string& relative_path) {
+#if BUILDFLAG(ENABLE_EXTENSIONS)
```

Comment:
sorry, I guess this entire file is only included for extensions so this buildflag isn't needed, but you are missing deps here for this file https://github.com/brave/brave-core/pull/9350/files#diff-1aa01bd2683612d12e2295206d1dd30e0460c9ec2ca3e9581a973836c2f2d4f9R49

Reply:
Removed buildflag and added deps.